### ISSUE TITLE
Raw what?

### ISSUE THREAD

**Issue by Sunconure11** [KnowYourKnot/Blockpacks#11]
![Screenshot 2021-03-06 13 31 56](https://user-images.githubusercontent.com/5614902/110217109-57ddcf00-7e80-11eb-82d5-fb561606a960.png)

**Comment by KnowYourKnot** [comment]
Lol, when mined it should appear as "Marble" with the tooltip text set to "Raw". I don't know if this is a Chisel Decor issue or just bizarre behaviour from HWYLA. What version of HWYLA are you using?

**Comment by jarn123** [comment]
maybe it says raw because i see  
"names": [
          "raw"
in the block pack

**Comment by jarn123** [comment]
i have a feeling that the whole language json file has to be redone

**Comment by jarn123** [comment]
![2021-03-08_17 03 15](https://user-images.githubusercontent.com/69686962/110346820-3de2ee80-8030-11eb-9276-e984559918e4.png)
i put that rawrfie.......... in the language folder soo RIP the whole language folder has to be redone

**Comment by jarn123** [comment]
![LUL emoji](https://user-images.githubusercontent.com/69686962/110347384-dda07c80-8030-11eb-8746-64b5baacab35.png)

**Comment by KnowYourKnot** [comment]
@jarn123 No, this is intentional. The tooltip of a Chisel Decor block is intended to appear like so:
![image](https://user-images.githubusercontent.com/40759698/110358608-843a4c80-8034-11eb-8914-4483fa876c2f.png)
Here, the actual name of the block is the type name (so, in this case, Marble) with a tooltip displaying the variant name (in this case, raw). The en_us.json file does not need to be edited in order to fix this issue. This is a compatibility issue with HWYLA.

**Comment by jarn123** [comment]
in my eyes you can just remove the marble thing below and set it to "Raw Marble" and hwyla does seem to listen to block.chiseldecor.classic.marble
